tests/test_dagfactory.py

```diff
@@ -61,6 +67,7 @@ def test_load_config_valid():
             },
         },
         "example_dag2": {
+            "doc_md_file_path" : "./fixtures/mydocfile.md",
```

Comment:
A few tests are failing with this error:

```
FileNotFoundError: [Errno 2] No such file or directory: './fixtures/mydocfile.md'
```

Do these need to be fully qualified? `os.path.join(here, "fixtures/mydocfile.md")`